cmake/RunTests.cmake

```diff
@@ -1,6 +1,13 @@
 # Set LC_ALL to meet expectations of some locale-sensitive tests.
 set(ENV{LC_ALL} "en_US.UTF-8")
 
+if(POLICY CMP0012)
+  # Avoid policy warning due to CI=true. This is needed even if the main
+  # project has already set this policy as policy settings are reset when using
+  # the cmake script mode (-P).
```

Comment:
More generally, `-P` inherits nothing, right? It's basically using cmake as a generic script engine (which is useful for cross-platform purposes).